app/src/main/java/org/wikipedia/notifications/NotificationActivity.kt

```diff
@@ -176,11 +183,26 @@ class NotificationActivity : BaseActivity() {
                     }) { t -> setErrorState(t) })
         }
 
+    private fun delimitedFilteredWikiList(): String {
+        val filteredWikiList = mutableListOf<String>()
+        if (Prefs.notificationsFilterLanguageCodes == null) {
+            filteredWikiList.add("*")
```

Comment:
Please add all wikis instead of adding `*` by default, otherwise, the user will not see notifications before actually setting languages in the filter page.

Reply:
Prefs.notificationsFilterLanguageCodes == null condition is only when they have not yet set any languages through filter @cooltey .. so "*" will get all wikis no?

Reply:
For my account, the `*` does not work properly (as we discussed in the Slack channel), which is a known issue. I would recommend using the current way I've been doing in the `wikiList()` as the default list. (or another way of fetching all `read` and `!read` notifications from all user' wikis.